### ISSUE TITLE
I'm having no luck getting model to learn another voice on different voicesets

### ISSUE THREAD

**Issue by MotorCityCobra** [NVIDIA/tacotron2#141]
I put this in the commandline with the path to my wavs  
sed -i -- 's,DUMMY,my_wav_dir/wavs,g' filelists/*.txt  

Is this where I go arry? Because I don't see this in the instructions.
I get an error unless I also put the txts of the train and val files in hparams so I set those

    training_files='filelists/jb_train.txt',
    validation_files='filelists/jb_val.txt'

I've been training on an M-AILABS Speech Dataset of audio books read by Judy_Bieber that is 15,700 utterances and it's not working out for me. Gibberish - BUT, the gibberish sounds like the Judy_Bieber voice and when I do a spot check of wav text and wav dirs by looking at the txt files they match.
I thought I replicated the training setup of the the LJ set but after 90,000 iterations it is gibberish just like it was at 1000 iterations. (The LJ set could speak fairly well at 90,000).

Before that I made a 46,000 utterance dataset combining Judy Bieber, Marry Ann and LJ. It trained for 160,000 interactions before the power went out and it never seemed to improve once. Also gibberish. BUT gibberish that was an melding of all three voices - what I wanted but I wanted a better trained model than the LJ one I trained to 360,000 iterations. At 360k it speaks simple sentences okay, but I want a TTS that can handle a curveball like the production level Google TTSs.

**Comment by rafaelvalle** [comment]
Make sure you remove silence from the beginning and end of audio files.
Also make sure your transcription is correctly aligned with the audio.
Combining multiple datasets should not be a problem.
What do your alignment plots look like?

**Comment by MotorCityCobra** [comment]
![76000_iterations](https://user-images.githubusercontent.com/14045554/52528611-cdcad880-2cb0-11e9-88dc-ca452b311de5.png)

I'm at 76000 iterations now, after about 3, maybe it was 4 days.
Spot checks of my transcripts look correct.

I trained on just the LJ set with all else being equal and it started to make sense at 90,000 iterations. Maybe 30,000 additional utterances will take longer.

I saw your other comment about ffmpeg to remove silence.
I converted to an array and removed the silent half second from each file kind of like this.  

    for i in wavs:
        new_wav = i[11025:-11025]
